solace-message-router/azuredeploy.parameters.json

```diff
@@ -2,20 +2,20 @@
   "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
-    "storageAccountName": {
-      "value": "GEN-UNIQUE"
-    },
     "adminUsername": {
       "value": "GEN-UNIQUE"
     },
     "adminPassword": {
       "value": "GEN-PASSWORD"
     },
-    "dnsLabelForPublicIP": {
+    "workspaceName": {
       "value": "GEN-UNIQUE"
     },
-    "solaceVMR_URI": {
-      "value": "http://em.solace.com/DCx500W0IBC00o01dY0V0i0"
+    "dnsLabelForLbIp": {
+      "value": "GEN-UNIQUE"
+    },
+    "dnsLabelForVmIp": {
+      "value": "GEN-UNIQUE"
     },
     "deploymentModel": {
```

Comment:
remove this and use a defaultValue in the template

Reply:
Removed